### ISSUE TITLE
Incorrect debug category log printing

### ISSUE THREAD

**Issue by thelazier** [dashpay/dash#582]
I found following lines are using incorrect debug category string.
```
$ git grep \"masnernode\"
src/masternode.cpp:    LogPrint("masnernode", "mnb - Accepted Masternode entry\n");
src/masternode.cpp:    LogPrint("masnernode", "CMasternodePing::CheckAndUpdate - New Ping - %s - %s - %lli\n", GetHash().ToString(), blockHash.ToString(), sigTime);
src/masternode.cpp:            LogPrint("masnernode", "CMasternodePing::CheckAndUpdate - Masternode ping accepted, vin: %s\n", vin.ToString());
src/masternode.cpp:        LogPrint("masnernode", "CMasternodePing::CheckAndUpdate - Masternode ping arrived too early, vin: %s\n", vin.ToString());
src/masternode.cpp:    LogPrint("masnernode", "CMasternodePing::CheckAndUpdate - Couldn't find compatible Masternode entry, vin: %s\n", vin.ToString());
src/masternodeman.cpp:        LogPrint("masnernode", "CMasternodeMan: Adding new Masternode %s - %i now\n", mn.addr.ToString(), size() + 1);
src/masternodeman.cpp:            LogPrint("masnernode", "CMasternodeMan: Removing inactive Masternode %s - %i now\n", (*it).addr.ToString(), size() - 1);
src/masternodeman.cpp:        LogPrint("masnernode", "mnp - Masternode ping, vin: %s\n", mnp.vin.ToString());
src/masternodeman.cpp:                LogPrint("masnernode", "dseg - Sending Masternode entry - %s \n", mn.addr.ToString());
src/masternodeman.cpp:            LogPrint("masnernode", "CMasternodeMan: Removing Masternode %s - %i now\n", (*it).addr.ToString(), size() - 1);
```

**Comment by UdjinM6** [comment]
fix is in #562

**Comment by thelazier** [comment]
really fast :+1:
